### DISCUSSION TITLE
文科女咋润。。已经读了两个硕士了。。后悔学文。。

### DISCUSSION THREAD

**Discussion by Wendy19990** [The-Run-Philosophy-Organization/run#318] [Q&A] [answered]
读的法学。。一个锅内学硕，一个hk llm...

**Comment by ytgui** [comment]
湾区老多单身的清北大佬了，不考虑一下？

（没有冒犯的意思，确实是个方法哈 :+1: :)

**Reply by Wendy19990** [reply]
> 湾区老多单身的清北大佬了，不考虑一下？
> 
> （没有冒犯的意思，确实是个方法哈 👍 人不在湾区。。人在大湾区。。😿

**Reply by the-run-philosophy** [reply]
对，找一个已经在美国的工程师单身男性拿到绿卡或者485的，联系好之后来美国结婚

**Comment by the-run-philosophy** [comment]
@Wendy19990 需要加群吗？或者微信

**Reply by the-run-philosophy** [reply]
你觉得需要这么一个平台吗？还是自己找就可以？我在想润学是否应该主动帮助这样的情况。提供联系方式，照片啥的。

**Reply by cambrian-rock** [reply]
需要🙋🏻

**Reply by Doris324** [reply]
需要🙋🏻‍♀️请问还可以加你吗？谢谢

**Reply by Wendy19990** [reply]
怎么加

**Reply by Zeroeisj** [reply]
需要请问怎么加?

**Reply by stackunderflow111** [reply]
给了联系方式和照片岂不是大大方便晶哥？

**Reply by cambrian-rock** [reply]
> 给了联系方式和照片岂不是大大方便晶哥？

是需要这个平台

**Comment by the-run-philosophy** [comment, marked as answer]
@Wendy19990 参考 https://github.com/The-Run-Philosophy-Organization/run/blob/main/%E6%B6%A6%E5%AD%A6%E6%96%B9%E6%B3%95%E8%AE%BA/%E5%90%84%E5%9B%BD%E9%80%89%E6%8B%A9/%E7%BE%8E%E6%B4%B2/%E7%BE%8E%E5%9B%BD/%E6%96%87%E7%A7%91%E5%A5%B3%E7%94%9F%E4%B8%8D%E5%A5%BD%E6%89%BE%E7%A8%8B%E5%BA%8F%E5%91%98%E7%9A%84%E5%B7%A5%E4%BD%9C%E5%A6%82%E4%BD%95%E6%B6%A6%E7%BE%8E%E5%9B%BD.md
